com/win32com/HTML/docindex.html

```diff
@@ -14,9 +14,10 @@ <H1>PythonCOM Documentation Index</H1>
 <P><A HREF="QuickStartServerCom.html">A Quick Start to Server Side COM</A></P>
 <P><A HREF="GeneratedSupport.html">Information on generated Python files (ie, what makepy generates)</A></P>
 <P><A HREF="variant.html">An advanced VARIANT object which can give more control over parameter types</A></P>
+<P><A HREF="COM_Records.html">COM Record Support</A></P>
 <P><A HREF="package.html">A brief description of the win32com package structure</A></P>
 <P><A HREF="PythonCOM.html">Python COM Implementation documentation</A></P>
-<P><A HREF="misc.html">Misc stuff I don�t know where to put anywhere else</A></P>
+<P><A HREF="misc.html">Misc stuff I don�t know where to put anywhere else</A></P>
```

Comment:
oops, encoding issue

Reply:
Damn, this encoding stuff is haunting us.
I didn't pay attention that the HTML has `charset=windows-1252` but VS Code had opened the file as UTF-8 encoded.
Interestingly Notepad++ opens it right away as Windows-1252 encoded and if you select to reopen the file with a different encoding also VS Code proposes Windows-1252 '_guessed from the content_'. Hmm, is there probably a setting in VS Code to open it right away with what it would guess?

However, I would propose a different fix, i.e. change:
`<META HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=windows-1252">`
to
`<META HTTP-EQUIV="Content-Type" CONTENT="text/html" charset="utf-8">`
in addition to
```suggestion
<P><A HREF="misc.html">Misc stuff I don't know where to put anywhere else</A></P>
```

What do you think?

Reply:
To quote @mhammond:
https://github.com/mhammond/pywin32/pull/2496#issuecomment-2727729700
> I'd be fine with making everyting utf-8 except there the encoding really does specifically matter.

So yeah, if these can be UTF-8, let's just remove the `charset=windows-1252` where we can. That was part of the goal in https://github.com/mhammond/pywin32/pull/2496 but I missed HTML files.

I also personally don't mind changing the apostrophe to a simpler single quote. Given how prevalent it is in code comments.

Reply:
OK, so I've also changed the encoding in the other HTML files and replaced the apostrophe with a single quote and the special ellipsis character with three consecutive dots where necessary. I did not use the unicode character _U+2026_ for the ellipsis because I didn't see any advantage in using a multibyte character here. At the end of the day these HTML pages will not win a web design contest.